packages-ts/gauntlet-solana-contracts/src/lib/time.ts

```diff
@@ -0,0 +1,39 @@
+import BN from 'bn.js'
```

Comment:
I guess something that could be useful for more than just Solana.

Reply:
Definitely, core should expose some BN library